tests/dev.yml

```diff
@@ -34,7 +34,7 @@ parameters:
         registry: dockerhub.vshn.net
 
     grpcEndpoint: 172.19.0.1:9443
-    proxyFunction: true
+    proxyFunction: false
```

Comment:
I think we agreed that this must remain in true

Reply:
oh ok. I missed that. I'll revert that then